src/utils.ts

```diff
@@ -10,7 +10,10 @@ import { ServerName } from './types'
  * @returns The url of that server
  */
 export async function getServerUrl(server: ServerName) {
-  const port = await getPort(server)
+  let port = await getPort(server)
+  if (process.env.LOCAL_DEV_SERVER === server) {
+    port = Number(process.env.LOCAL_DEV_PORT) || 3000
+  }
```

Comment:
```ts
const hasLocalServer = process.env.LOCAL_DEV_SERVER === server
const port = hasLocalServer ? Number(process.env.LOCAL_DEV_PORT || 3000) : await getPort(server)
```

just to avoid mutating as much as possible :P

Reply:
updated!